app/views/ad_hoc_data_sources/download.xlsx.axlsx

```diff
@@ -0,0 +1,8 @@
+wb = xlsx_package.workbook
+csv = GrdaWarehouse::AdHocDataSource.blank_csv
+
+wb.add_worksheet(name: 'Template'.slice(0, 30)) do |sheet|
+  title = sheet.styles.add_style(sz: 12, b: true, alignment: {horizontal: :center})
+  sheet.add_row(csv.first.split(','), style: title)
+  sheet.add_row(csv.last.split(','))
```

Comment:
I'd make `blank_csv` an array tp start with instead of splitting on a comma... And if, there might be more than one row, I'd like there to be an `each` -- probably not the case here, but I'd worry about adding data to `blank_csv` and wondering why it doesn't appear in the XLSX.